st2actions/conf/syslog.conf

```diff
@@ -18,15 +18,11 @@ formatter=syslogVerboseFormatter
 args=()
 
 [handler_auditHandler]
-class=st2common.log.FormatNamedFileHandler
+class=st2common.log.ConfigurableSyslogHandler
 level=AUDIT
-formatter=jsonFormatter
-args=('logs/st2actionrunner.{pid}.audit.log',)
```

Comment:
We have a separate file for writing audit logs. Are you sure this change is valid? Are you seeing an audit log?